packages/dates/src/date-time-utils/DateTimeUtils.ts

```diff
@@ -20,12 +20,12 @@ const DATE_DISPLAY_VARIANTS: Readonly<
   Record<DateDisplayVariant, Intl.DateTimeFormatOptions>
 > = {
   Date: {
-    day: '2-digit',
+    day: 'numeric',
     month: 'short',
     year: 'numeric',
   },
   ShortDate: {
-    day: '2-digit',
+    day: 'numeric',
```

Comment:
Should we update it for DateTime as well?

Reply:
I think no need, but I will try it there as well

Reply:
fixed DateTime day property moved to numeric